### PR TITLE
Refactoring and test

### PR DESCRIPTION
**Pull request by adrissonGalvao**

### CODE REVIEW

File: service/sensor.go [hunk 1]
```diff
@@ -12,13 +12,20 @@ import (
...
-func init() {
-	sensorRepository.Connect()
+type ISersoService interface {
+	Create(w http.ResponseWriter, r *http.Request)
+	FindAll(w http.ResponseWriter, r *http.Request)
+	FindOne(w http.ResponseWriter, r *http.Request)
+	Delete(w http.ResponseWriter, r *http.Request)
+	Update(w http.ResponseWriter, r *http.Request)
+	VerifyIntegrityUrlSensor(id string) bool
+	GeneratingUrlSensors() ([]string, error)
```
Comment (edvmorango) [inline service/sensor.go]: @adrissonGalvao please rename  "GeneratingUrlSensors", this name don't fit in your names pattern.

File: service/sensor.go [hunk 2]
```diff
@@ -12,13 +12,20 @@ import (
 	"gopkg.in/mgo.v2/bson"
 )
 
-var sensorRepository = repository.SensorRepository{}
-
-func init() {
-	sensorRepository.Connect()
+type ISersoService interface {
```
Comment (edvmorango) [inline service/sensor.go]: @adrissonGalvao the service name is really "Serso"?

**Review by edvmorango — CHANGES_REQUESTED** [review]

File: service/sensor.go (hunk 1 above)
Comment (adrissonGalvao) [reply service/sensor.go]: fixed it

**Review by edvmorango — APPROVED** [review]
Pr Approved

**Pull request merged**